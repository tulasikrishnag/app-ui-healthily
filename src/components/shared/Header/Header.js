import React from "react";
import { makeStyles } from "@material-ui/core/styles";
import { AppBar, Grid } from "@material-ui/core";
import Toolbar from "@material-ui/core/Toolbar";
import Typography from "@material-ui/core/Typography";

const useStyles = makeStyles((theme) => ({
  bg: {
   backgroundColor:"#FF8000"
  },
  tg:
  {
    color: "#1C1C1C"
  }
}));

export default function Header() {
  const classes = useStyles();
  return (
    <>
      <Grid
        container
        spacing={0}
        direction="column"
        alignItems="center"
        justify="center"
      >
        <AppBar className={classes.bg}>
          <Toolbar>
            <Typography variant="h4" className={classes.tg}>
              Healthily
            </Typography>
          </Toolbar>
        </AppBar>
      </Grid>
    </>
  );
}
