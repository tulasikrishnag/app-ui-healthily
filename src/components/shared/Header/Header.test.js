import { render } from "@testing-library/react";
import Header from "./Header";
import React from 'react'
describe("Header", () => {
  it("render component as expected", () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});
