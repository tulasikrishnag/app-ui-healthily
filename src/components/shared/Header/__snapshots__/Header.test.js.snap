// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header render component as expected 1`] = `
<DocumentFragment>
  <div
    class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
  >
    <header
      class="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary makeStyles-bg-1 mui-fixed MuiPaper-elevation4"
    >
      <div
        class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
      >
        <h4
          class="MuiTypography-root makeStyles-tg-2 MuiTypography-h4"
        >
          Healthily
        </h4>
      </div>
    </header>
  </div>
</DocumentFragment>
`;
